src/controllers/Items.js

```diff
@@ -42,6 +43,23 @@ export default {
     });
   },
 
+  incrementUsage(req, res) {
+    const itemId = req.params.id;
+
+    db.Item.update({ delimiter: Sequelize.literal('delimiter + 1') }, { where: { itemId } })
+      .then((response) => {
+        const increment = response[0];
+
+        res.status(200).json({
+          message: 'success',
```

Comment:
If the message is supposed to be used in the client, maybe a better response text is wanted, but otherwise this should be ok!